cli/src/main/java/com/devonfw/tools/ide/tool/intellij/Intellij.java

```diff
@@ -68,6 +63,56 @@ protected void postInstall() {
     EnvironmentVariables envVars = this.context.getVariables().getByType(EnvironmentVariablesType.CONF);
     envVars.set("IDEA_PROPERTIES", this.context.getWorkspacePath().resolve("idea.properties").toString(), true);
     envVars.save();
+    createStartupCommandScript();
+  }
+
+  private void createStartupCommandScript() {
+    this.context.getFileAccess().mkdirs(getToolPath().resolve("bin"));
+    Path bashFile;
+    String bashFileContentStart = "#!/usr/bin/env bash\n'";
+    String bashFileContentEnd = "' $*";
+    if (this.context.getSystemInfo().isMac()) {
+      try {
+        bashFile = Files.createFile(getToolBinPath().resolve(getName()));
+        Files.writeString(bashFile,
+            bashFileContentStart + getToolPath().resolve("IntelliJ IDEA CE.app").resolve("Contents").resolve("MacOS").resolve(IDEA) + bashFileContentEnd);
+      } catch (IOException e) {
+        throw new RuntimeException(e);
+      }
+      // Setting execute permissions is only required if executed on a real MacOS, won't work on Windows.
+      if (SystemInfoImpl.INSTANCE.isMac()) {
```

Comment:
Must also be done on Linux.
Why is the method called `setMacOsFilePermissions`.
I commented about this topic in several PRs already.
IMHO we need a `setExecutable` or `makeExecutable` method in `FileUtil` that does `chmod a+x` on a given file and will do nothing on Windows (and not fail there).

Reply:
Created issue #542

Reply:
I've implemented the changes of #542 here too now.

Reply:
Excellent solution. 👍